packages/story-editor/src/app/pageDataUrls/pageDataUrlsProvider.tsx

```diff
@@ -16,22 +16,24 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
+import type { Page } from '@googleforcreators/elements';
 import { useMemo, useCallback, useState } from '@googleforcreators/react';
+import type { PropsWithChildren } from 'react';
 
 /**
  * Internal dependencies
  */
 import useIdleTaskQueue from '../../utils/useIdleTaskQueue';
 import storyPageToDataUrl from '../../utils/storyPageToDataUrl';
 import Context from './context';
+import type { PageDataUrlsState, PageDataUrlsActions } from './types';
 
-/**
- * @typedef {import('../../types.js').Page} Page
- */
-
-function PageDataUrlProvider({ children }) {
-  const [dataUrls, setDataUrls] = useState({});
+function PageDataUrlProvider({
+  children,
+}: {
+  children: PropsWithChildren<Record<string, never>>;
```

Comment:
See [this for context](https://github.com/GoogleForCreators/web-stories-wp/pull/12744#discussion_r1033995957).
```suggestion
  children: PropsWithChildren<unknown>;
```